src/services/chris_integration.tsx

```diff
@@ -5,6 +5,7 @@ import { DcmImage } from "../context/reducers/dicomImagesReducer";
 import DicomViewerService from "../services/dicomViewerService";
 import { PluginModels } from "../api/app.config";
 import { formatTime, modifyDatetime } from "../shared/utils"
+import { Processing } from '../services/pastAnalysisService';
```

Comment:
Not sure if we want to refactor this constant to chris_integration? Seems a bit of bad code design to be importing an enumeration each time from `pastAnalysisService`, especially if other parts of the code may want to use it in future too

Reply:
Good suggestion! I'll actually move this enum to `Types.tsx`

Reply:
I'm not sure if it'll be ideal to move it to `chirs_integration`, since it's also being used in `pastAnalysisService` and `PastAnalysisTable`. Do you think we should create a new file to store enums like this?

Reply:
Is there a reason why this is an enum? Seems like that enum contains only 1 value, which is "AnalysisAreProcessing".

From a brief look at the usages, I am not even sure if we need that enum/constant at all. It seems like we are just using it as a flag to determine where an analysis is in the middle of processing or not. If that's the case, we might as well use an empty string for that. The only advantage is that the constant explicitly tells that something is processing, while empty string would infer that it is processing.

``` 
if (pluginStatus !== PluginPollStatus.SUCCESS &&
  pluginStatus !== PluginPollStatus.ERROR &&
  pluginStatus !== PluginPollStatus.CANCELLED) {
  analysisCreated = Processing.analysisAreProcessing;
} else {
  analysisCreated = modifyDatetime(findDircopy.data.start_date);
}
```

Reply:
Agreed. It'll be cleaner to just stick with using an empty string.